data/ut/legislature/Derrin-R-Owens-9143a45a-9d2e-4c4e-8ea8-e6204f0ee959.yml

```diff
@@ -0,0 +1,59 @@
+id: ocd-person/9143a45a-9d2e-4c4e-8ea8-e6204f0ee959
+name: Derrin R. Owens
+email: derrinowens@le.utah.gov
+image: https://le.utah.gov/images/legislator/OWENSD.jpg
+party:
+- name: Republican
+roles:
+- type: upper
+  jurisdiction: ocd-jurisdiction/country:us/state:ut/government
+  district: '27'
+offices:
+- classification: district
+  address: P.O. Box 127, Fountain Green, UT, 84632
+links:
+- url: https://le.utah.gov/asp/billsintro/SenResults.asp?Listbox2=OWENSD
+  note: legislation
+- url: https://le.utah.gov/Documents/demographic/profiles/2022/Senate_Dist27.pdf
```

Comment:
@sroomf this person should already have a file, we just need to add their new role:
- type: upper
  jurisdiction: ocd-jurisdiction/country:us/state:ut/government
  district: '27'

They served in senate district 24 from Jan 2021-Jan2023 and in House District 58 from August 2015-Jan 2021